emb03/acquia_cms
# ACMS-1073: Robust Capabilities

**Motivation**

Adds new module that provides a low-barrier of entry to get up and running with a NEXTjs site by automating many steps required to get a site working.

**Proposed changes**
• When installing Robust module, create a user named "Headless" with a role of "headless" and a customized consumer is created
• Customized consumer displays: User, Description, UID, secret, Scopes and Operations

**Testing steps**
 Install a new site
• Enable the ACMS Headless module and Robust Headless module
• Go to the consumer page, new consumer should be appear
• Go to next entity page, next site should appear
• Go to consumer edit page User (Headless) and Scope (headless) should be filled out and editable.
• Go to admin/people a new user should be there with the name of "Headless" and role of "headless".
• Uninstall the robust module, go to admin/people user (Headless) with role (headless) should have remained
• Go to consumer page, consumer should remain
• Go to next entity page, next site should remain
• Re-install Headless Robust module, you should not get errors

**Merge requirements**
